CMakeLists.txt

```diff
@@ -29,16 +29,17 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 # Location of .pycodestyle for norm checking within IODA-converters
 set( IODACONV_PYLINT_CFG_DIR ${CMAKE_CURRENT_SOURCE_DIR} )
 
+find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
+
 set(pyver python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR})
-set(PYDIR ${pyvers}/pyiodaconv)
+set(PYDIR ${pyver}/pyiodaconv)
 
 # Location of installed python iodaconv libraries
 set( PYIODACONV_BUILD_LIBDIR   ${CMAKE_BINARY_DIR}/lib/${PYDIR} )
-set( PYIODACONV_INSTALL_LIBDIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/${PYDIR} )
+set( PYIODACONV_INSTALL_LIBDIR ${CMAKE_INSTALL_PREFIX}/lib/${PYDIR} )
```

Comment:
I've noticed that `<install-prefix>/lib/python<version>/site-packages` seems to be, typically, where setup.py, spack-stack and pip install user built packages. Should we follow that convention here?

What do others think about using the site-packages location?I can go either way (what is in this PR vs site-packages).

I think what is much more important is that we eventually end up installing all ioda-converters and ioda python packages in the same place and that place be findable without having to use PYTHONPATH settings.

Reply:
@CoryMartin-NOAA @srherbener I checked the Python way of doing this. If you use pip install or there ilk, things will be installed in the site-packages directory of PYTHONHOME. This directory is always in PYTHONPATH unless you clobber this variable it which is not advised at all. There no reason to set site-packages in cmake since it comes for free with a pip install and well it might be confusing to most outside python users. So it looks like we are doing it the "python way"

Reply:
Thanks for following up on this!